great-idea-website/css/index.css

```diff
@@ -58,9 +58,99 @@ html, body {
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
+    font-family: 'Barriecito', cursive;
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body {
+	margin: 0 25%;
+}
+
+}
+.Navigation {
+	background: green
+}
+nav a {
+	
+	margin: 3%;
+}
+
+nav img {
+	margin: 0 0 0 7%;
+}
+
+.CTALeft {
+	display: inline-block;
+	margin: 0 0 0 10%;
+	}
+
+.CTARight {
+	display: inline-block;
+	margin: 10% 0 0 33%;
+}
+
+header h1 {
+	text-align: center;
+	width: 60px;
+	font-size: 80px;
+}
+
+header p {
+	font-size: 20px;
+	padding: 6px 20px;
+	text-align: center;
+	border: 1px solid black;
+
+ }
+
+
+.topcontent {
+	border-top: 1px solid black;
+	margin: 60px 50px 40px 50px;
+}
+.Features {
+	margin: 3%;
+	display: inline-block;
+	width: 43%;
+}
+
+.About {
+	margin: 3%;
+	display: inline-block;
+	width: 43%;
+}
+.middle-img {
+	float: right;
+}
+.bottomcontent {
+	margin: 20px;
+	border-bottom: 1px solid black;
+}
+
+.Services {
+	margin: 3%;
+	display: inline-block;
+	width: 27%;
+}
+
```

Comment:
The reason your sections don't fit in a row is because if you add up all your margins and widths they exceed the width of the page. you can try playing around with the numbers here try using either just percentages or pixels but not a combination of both.